plugins/domains/commands

```diff
@@ -46,7 +46,7 @@ case "$1" in
       fi
       if [[ "$VHOST" =~ $RE_IPV4 ]] || [[ "$VHOST" =~ $RE_IPV6 ]];then
         echo "unsupported vhost config found. disabling vhost support"
-        [[ ! $(grep -q NO_VHOST "$DOKKU_ROOT/$APP/ENV") ]] && echo "export NO_VHOST='1'" >> "$DOKKU_ROOT/$APP/ENV"
+        [[ ! $(grep -q NO_VHOST "$DOKKU_ROOT/$APP/ENV") ]] && dokku config:set-norestart $APP NO_VHOST=1
```

Comment:
Can we remove the grep? The `config:set-norestart` will force the correct value in all cases I would think.